helmcharts/notes/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "notes.fullname" . }}-test-connection"
+  labels:
+{{ include "notes.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
```

Comment:
In other specs we specify a version of the busybox image.  Should we specify here as well?

Reply:
done